src/analysis/retail/evoker/preservation/normalizers/CastLinkNormalizer.ts

```diff
@@ -464,9 +491,18 @@ const EVENT_LINKS: EventLink[] = [
     backwardBufferMs: 50,
     forwardBufferMs: 50,
     additionalCondition(linkingEvent, referencedEvent) {
-      return HasRelatedEvent(linkingEvent, LIFEBIND_APPLY); // make sure the heal is on someone with lifebind buff
+      return HasRelatedEvent(linkingEvent, LIFEBIND); // make sure the heal is on someone with lifebind buff
     },
   },
+  {
+    linkRelation: GOLDEN_HOUR,
+    linkingEventId: SPELLS.GOLDEN_HOUR_HEAL.id,
+    linkingEventType: EventType.Heal,
+    referencedEventId: [TALENTS_EVOKER.REVERSION_TALENT.id, SPELLS.REVERSION_ECHO.id],
```

Comment:
Adding the non-echo spellid here is unnecessary since you filter it out in 100% of scenarios later on anyway, unless you are planning on/are using this event link for something else

Reply:
Yeah I just included it in case it's needed in the future